plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/checkpoint/CheckpointWriter.java

```diff
@@ -51,6 +51,7 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
+import java.util.function.Consumer;
 
```

Comment:
Please update commit message to
`Avoid failures from large Pages when writing delta checkpoint`

Reply:
addressed